### ISSUE TITLE
Less hint breaks with Media query variables

### ISSUE THREAD

**Issue by srogers202** [lesshint/lesshint#64]
Lesshint fails with media query variables like this:
```
@tablet:    ~"only screen and (min-width: 720px) and (max-width: 959px)";
```

Error:
```
gonzales-pe: Please check validity of the block starting from line #
```

**Comment by jwilsson** [comment]
It looks like you ran into [#17](https://github.com/tonyganch/gonzales-pe/issues/17) over at `gonzales-pe`, hopefully we'll have it fixed soon.

**Comment by jwilsson** [comment]
This should've been fixed in `1.1.0`.

Please reopen if you still run in to it.
